app/controllers/api/v1/emotions_controller.rb

```diff
@@ -73,4 +77,12 @@ def emotion_params
   def time_period
     @time_period ||= TimePeriod.find_or_create_time_period
   end
+
+  def list_shoutouts_to(user)
+    shoutouts_to_user = ShoutoutRecipient.where(user_id: user.id)
+    return {} if shoutouts_to_user.empty?
+    return Shoutout.where(id: shoutouts_to_user[0].shoutout_id) if shoutouts_to_user.length == 1
```

Comment:
you can completely rewrite this method and simplify it when you use new models relations, for example you can get all shoutouts for users use `current_user.mentions` if you do right configuration of your models

Reply:
Thanks I rewritten it before